benchmarks/graphics_pipeline/main.cpp

```diff
@@ -137,10 +165,11 @@ class ProjApp
     grfx::DescriptorSetLayoutPtr                                  mSetLayout;
     std::array<grfx::ShaderModulePtr, kAvailableVsShaders.size()> mVsShaders;
     std::array<grfx::ShaderModulePtr, kAvailablePsShaders.size()> mPsShaders;
-    PerspCamera                                                   mCamera;
+    std::unique_ptr<FreeCamera>                                   mCamera;
```

Comment:
Why is this a pointer? Isn't a simple member enough since the lifetime of the camera is tied to the lifetime of the app?
Seems like the constructor could be called in the app constructor.